README.md

```diff
@@ -13,3 +9,31 @@ Make sure that you have the compatible versions of Visual Studio 2019 and .NET C
+
+## Related links
+[Learn More about Blazor Date Picker](https://www.syncfusion.com/blazor-components/blazor-datepicker?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples)<br/><br/>
+[Documentation](https://blazor.syncfusion.com/documentation/datepicker/getting-started?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples)<br/><br/>
+[Download Free Trial](https://www.syncfusion.com/downloads/blazor-samples?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples)<br/><br/>
+[Release History](https://blazor.syncfusion.com/documentation/release-notes/19.4.38?type=all?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples)<br/><br/>
+[Online Demos](https://blazor.syncfusion.com/demos/datepicker/default-functionalities?theme=fluent?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples)<br/><br/>
+[Contact Support](https://www.syncfusion.com/support/directtrac/incidents/newincident/?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples)<br/><br/>
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples)<br/><br/>
+[Community Forums](https://www.syncfusion.com/forums?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples)<br/><br/>
+[Knowledge base](https://www.syncfusion.com/kb?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples)<br/><br/>
+[Pricing](https://www.syncfusion.com/sales/products/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples)
+
+## About Syncfusion Blazor UI Controls
+Syncfusion's [Blazor UI Controls](https://www.syncfusion.com/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples) library is the only suite that you will ever need to build an application since it contains over 100 high-performance, lightweight, modular, and responsive UI Controls in a single package. In addition to DatePicker component, we provide popular Blazor Controls such as [DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples), [Charts](https://www.syncfusion.com/blazor-components/blazor-charts?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples), [Diagram](https://www.syncfusion.com/blazor-components/blazor-diagram?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples), and [Scheduler](https://www.syncfusion.com/blazor-components/blazor-scheduler?utm_source=github&utm_medium=listing&utm_campaign=blazor-date-picker-github-samples).
```

Comment:
Seems you have the content edited from desktop platform. Refer any web platform and change it. 

Mentioned 100 controls, change it

Reply:
updated